src/query/string_opfunc.c

```diff
@@ -25033,9 +25034,13 @@ db_get_like_optimization_bounds (const DB_VALUE * const pattern, DB_VALUE * boun
 	}
       else
 	{
+	  /* Looking for an upper bound, therefore there is an LT compare.  */
+	  db_value_set_compare (bound, 1);
```

Comment:
Setting the compare_LT to false is not necessary. It is its default state. It is only set to true when it is meant to be true.